packages/reports/tests/acceptance/date-filter-test.js

```diff
@@ -65,4 +65,66 @@ module('Acceptance | date filter', function(hooks) {
       .dom('.filter-values--dimension-date-range-input__high-value .dropdown-date-picker__trigger')
       .includesText('05,', 'The high value is still set after the report is saved');
   });
+
+  test('verify the different time grains work as expected', async function(assert) {
```

Comment:
Can we add an `assert.expect` statement since we've got some async stuff going on in this test?